backend/models/indUserModel.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose')
+
+const validateEmail = (email) => {
```

Comment:
So me and Ryan looked at the validation rules and Ryan suggested we should use Mongoose validations (reference check out Ryan's branch) if you follow the OrgSchema, the codes are much cleaner :grimacing:

Reply:
We'll take a look and refactor once Ryan's has been merged into main, as we'll be able to compare a little easier that way